README.md

```diff
@@ -1,55 +1,62 @@
 # ed25519-WASM
-A drop-in replacement for `golang/crypto/ed25519` ([godoc](https://godoc.org/golang.org/x/crypto/ed25519), 
-[github](https://github.com/golang/crypto/tree/master/ed25519))
- with additional functionality compiled to WASM
 
-# Motivation
+A drop-in replacement for `crypto/ed25519` ([pkg.go.dev](https://pkg.go.dev/crypto/ed25519), [github](https://github.com/golang/crypto/tree/master/ed25519))
+with additional functionality compiled to WASM.
```

Comment:
This isn't quite right. `spacemeshos/ed25519` is the drop in replacement, but this library only contains the additional functions for singing and verification that create signatures that can be used for key extraction.